packages/capture/src/builder/DefaultHarBuilder.ts

```diff
@@ -0,0 +1,339 @@
+import { CaptureHar } from '../types/capture';
+import { BuilderEntryParams, HarBuilder } from './HarBuilder';
+import { Parser } from '../parser';
+import { transformBinaryToUtf8 } from '../utils';
+import * as Har from 'har-format';
+import * as Request from 'request';
```

Comment:
Don't use wildcard imports, you should explicitly import the things you want to use in a module

```suggestion
import HAR from 'har-format';
import request from 'request';
```